types/node/ts4.0/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+// TODO: duplicate or extend?
+{
+    "files": [
+        "index.d.ts",
+        "node-tests.ts"
+    ],
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "esnext",
```

Comment:
After I copied this from `types/node/tsconfig.json`, this was `es2018` but I had errors due to usage of `bigint`. I copied this line from `types/node/ts3.6/tsconfig.json`.